www-src/app/templates/permissions.jade

```diff
@@ -4,7 +4,7 @@
     .card.no-shadow.more-spacing.flat
         .item.item-text-wrap
             p
-                | Votre terminal requiert les permissions suivantes
+                | t('permissions explanations')
```

Comment:
@nono it's already here : https://github.com/cozy/cozy-mobile/blob/development/www-src/app/locales/en.json#L68 :smile: